### PR TITLE
Added a new attribute to require HTTPS on Web API endpoints

### PR DESCRIPTION
**Pull request by AnthonyMacKinnon**

### CODE REVIEW

**Comment by dalie** [comment]
LGTM

File: src/Georadix.WebApi.Tests/RequireHttpsAttributeFixture.cs [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            {
+                return new object[][]
+                {
+                    new object[] { HttpMethod.Delete, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Get, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Head, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Post, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Put, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Delete, false, HttpStatusCode.NotFound },
+                    new object[] { HttpMethod.Get, false, HttpStatusCode.Redirect },
```
Comment (StephaneAlie) [inline src/Georadix.WebApi.Tests/RequireHttpsAttributeFixture.cs]: Curious, what's reason behind redirecting 'GET' and 'HEAD' but not 'DELETE' for example?

**Comment by StephaneAlie** [comment]
LGTM

File: src/Georadix.WebApi.Tests/RequireHttpsAttributeFixture.cs (hunk 1 above)
Comment (AnthonyMacKinnon) [inline src/Georadix.WebApi.Tests/RequireHttpsAttributeFixture.cs]: Most HTTP stacks will redirect automatically for GET and HEAD requests, but not others.

**Pull request merged**